charts/kong/Chart.yaml

```diff
@@ -1,20 +1,19 @@
 apiVersion: v2
 description: The Cloud-Native Ingress and API-management
-engine: gotpl
 home: https://konghq.com/
 icon: https://s3.amazonaws.com/downloads.kong/universe/assets/icon-kong-inc-large.png
 maintainers:
-- name: hbagdi
-  email: harry@konghq.com
-- name: rainest
-  email: traines@konghq.com
+  - name: hbagdi
```

Comment:
Let the indentation style wars begin

Reply:
Hehe, I made it consistent with the other one and I admit - my editor defaults